.github/workflows/auto-maintenance.yml

```
@@ -1,4 +1,4 @@
-name: Coverage pages
+name: Automatic maintenance
```

```suggestion
name: Automatic Maintenance
```

I'd recommend title case for the name

---

Why would you capitalize that?

---

To comply with [title case](https://en.wikipedia.org/wiki/Title_case). It's good fashion to use it in scenarios like this